strax/processing/peak_properties.py

```diff
@@ -100,3 +100,52 @@ def compute_widths(peaks, select_peaks_indices=None):
     i = len(desired_fr) // 2
     peaks['width'][select_peaks_indices] = fr_times[:, i:] - fr_times[:, ::-1][:, i:]
     peaks['area_decile_from_midpoint'][select_peaks_indices] = fr_times[:, ::2] - fr_times[:, i].reshape(-1,1)
+
+@export
+#@numba.njit(nopython=True, cache=True)
```

Comment:
Hi Daniel
Thanks for checking at this, the functions calculate different things
compute_index_of_fraction() return the index at which certain area has been achieved if the fraction is 1 it will return peak['length'] so this is in units of samples.

compute_wf_attributes() for a given set of quantiles (fractions) it return the the amount of time elapsed for a given fraction of the total waveform area to be observed

So in other words, compute_index_of_fraction() is in units of samples and compute_wf_attributes() is in units of ns

Reply:
also `compute_wf_attributes()` can handle an array of peaks whereas `compute_index_of_fraction()` can do only peak by peak
Do you think it would better to merge into one function?

Reply:
Looking at these codes, `compute_index_of_fraction` can also return floats. I think it is better to utilize `compute_index_of_fraction` or modify it.
https://github.com/AxFoundation/strax/blob/9b508f7f8d441bf1fe441695115d292c59ce631a/strax/processing/peak_properties.py#L48